src/components/clusters/ClusterStatus.tsx

```diff
@@ -37,6 +37,8 @@ const getStatusIcon = (status: Cluster['status']): React.ReactElement => {
       return <CheckCircleIcon color={okColor.value} />;
     case 'finalizing':
       return <InProgressIcon />;
+    case 'adding-hosts':
```

Comment:
All states with the same icon can be collapsed:
```
case 'preparing-for-installation':
case 'installing':
case 'finalizing':
case 'adding-hosts':
      return <InProgressIcon />;
```

Reply:
done